tools/regions.py

```diff
@@ -116,6 +117,7 @@ def merge_region_list(
     destination - file name to write all regions to
     padding - bytes to fill gaps with
     restrict_size - check to ensure a region fits within the given size
+    pad_to_restrict_size - fill with padding up to restrict_size
```

Comment:
I do not think we need `pad_to_restrict_size`, I cannot see a situation where this would be needed and without it I think it fixes the regression that appeared in 11.1